modules/apps/transfer/keeper/keeper.go

```diff
@@ -156,3 +158,20 @@ func (k Keeper) AuthenticateCapability(ctx sdk.Context, cap *capabilitytypes.Cap
 func (k Keeper) ClaimCapability(ctx sdk.Context, cap *capabilitytypes.Capability, name string) error {
 	return k.scopedKeeper.ClaimCapability(ctx, cap, name)
 }
+
+// GetDenomHash retreives the hash key from the store for a demon trace
+func (k Keeper) GetDenomHash(ctx sdk.Context, denomTrace types.DenomTrace) (tmbytes.HexBytes, bool) {
```

Comment:
Maybe to group things that are related together: would it be better to move this function up and put it just before `AuthenticateCapability`?

Reply:
sure